injectdll/injector.py

```diff
@@ -15,13 +43,13 @@ def __init__(self, pid, backend_name, dll_name, is_unicode=False):
         """Constructor inject dll (one application - one class instance)."""
         self.is_unicode = is_unicode
         self.pid = pid
-        if not sysinfo.is_x64_Python() == is64bitprocess(self.pid):
+        if not sysinfo.is_x64_python() == sysinfo.is64_bitprocess(self.pid):
             raise RuntimeError("Application and Python must be both 32-bit or both 64-bit")
         self.h_process = self._get_process_handle(self.pid)
 
         self.dll_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                     'backends', backend_name,
-                                     'bin', 'x{}'.format("64" if is64bitprocess(self.pid) else "86"),
+                                     '../libs', backend_name,
```

Comment:
It's better to use `os.path.dirname(...)` than using `../` which may be platform specific.

Reply:
Ok, will be fixed in next commits